Catappult/appcoins-wallet-android
# [APPC-4043] No internet connection messages

**What does this PR do?**

Add feedback to user that they lose internet.

**Database changed?**

No

**How should this be manually tested?**

  on Home and Rewards, turn off internet and see a no internet snack bar appears, on other flows, turn off internet and try to do something, internet error should appear, on iap you should see a card on top of the dialog 

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4043](https://aptoide.atlassian.net/browse/APPC-4043)

**Code Review Checklist**



[APPC-4043]: https://aptoide.atlassian.net/browse/APPC-4043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
